### DISCUSSION TITLE
Do I need to translate my academic evidence for the GitHub Education application?

### DISCUSSION THREAD

**Discussion by Gitomochang** [community/community#134496] [GitHub Education]
### Hi everyone,

My application for GitHub Education was rejected for the following reasons.
- Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information. If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.
- Please ensure that your academic affiliation document contains your first name exactly as it appears in your GitHub billing information. Variations of your name and nicknames are unacceptable.

However, my billing information and student ID information are exactly matched. So I thought it might be due to the fact that the student ID card is written entirely in Japanese except for the name of the university. 

FAQ says to take a photo of the original piece of evidence next to a typed English translation, but how exactly should I typed my translation?
- Should I make a list of the Japanese-written information and its English translation?
- Does the font size or position of the translated text must exactly match that of the original student ID card?
- How do I place the typed translation and the original student ID card so the system will read it accurately?

**Comment by hubertit** [comment]
Hi there,

Yes, you should translate your academic evidence. Create a list of the Japanese-written information with its English translation. The font size and position don't need to match exactly. Take a clear photo of your student ID next to the translation on a single sheet of paper. Ensure everything is legible.

Good luck!
